stage/one/s9_if.py

```diff
@@ -0,0 +1,56 @@
+from utils.form import blank_form
+from utils import fields_generate
+
+
+route = {
+    'type': blank_form,
+    'url': '/stage/one/if',
+    'methods': ['GET', 'POST']
+}
+
+data = {
+    'title': 'Python Strings',
+    'description': [
+        'Lets get familiar with Python If-Conditions!',
+        'As the name indicates, If-conditions work as' +
+        'they are named, "If something happened, then something else happens".',
+        'You can reference if statements here: https://docs.python.org/3/' +
+        'tutorial/controlflow.html',
+        '\n',
+        'Now lets work on some exercises. A person is 20 year old age' +
+        'is she young or old?',
+        'Create a if statement that prints that if a person is older than' +
+        '18 years old is old, otherwise the person is young.'
+    ],
+    'code': [
+        'age=20',
+        'if(_____):',
+            'print(\'You are old\')',
+        'else:'
+            'print(\'You are young\')',
+    ],
```

Comment:
The code [here](https://github.com/claudiavmbrito/PyFun-Backend/blob/master/stage/one/s9_if.py#L25,#L31) are wrong.
It sould be like:
```python
'code': [
        'age=20',
        'if(_____):',
        '    print(\'You are old\')',
        'else:'
        '    print(\'You are young\')',
    ],
```
Don't forget the sapce!